### PR TITLE
rework logging, use separate patterns for stdout and file

### PR DESCRIPTION
**Pull request by matikij**

### CODE REVIEW

File: bin/autosign [hunk 1]
```diff
@@ -141,10 +141,12 @@ pre do |global,command,options,args|
   @logger.level = :info if global['verbose']
   @logger.level = :debug if global['debug']
 
-  if global['logfile'].nil?
-    @logger.add_appenders Logging.appenders.stdout
-  else
-    @logger.add_appenders Logging.appenders.stdout, Logging.appenders.file(global['logfile'])
+  stdout_layout = Logging.layouts.pattern(pattern: "%-5l -- %c : %m\n")
```
Comment (danielparks) [inline bin/autosign]: I think everywhere else uses `=>`. Mind changing those?

**Comment by danieldreier** [comment]
Thank you for contributing!

It looks to me like CI is failing on account of the `tins` gem requiring ruby 2.0.0 or greater. I'm comfortable removing ruby 1.9.3 support in CI, because it's been unsupported for some time. 

Would you mind updating [.travis.yml](https://github.com/danieldreier/autosign/blob/master/.travis.yml#L7:L8) and removing `1.9.3` and `jruby-19mode` so that this PR passes CI before we merge?

**Comment by danielparks** [comment]
@danieldreier I created https://github.com/danieldreier/autosign/pull/29

**Comment by danieldreier** [comment]
merged, thanks @danielparks

**Comment by danielparks** [comment]
Oh, sorry, I'm holding off on merging because of the license thing. I left this on your other PR too:

> @danieldreier is trying to switch the license to Apache… would you mind giving a +1 on #24 to indicate that you're okay with using the Apache license?
> 
> Thanks for these patches!

**Comment by danielparks** [comment]
Thanks much!

**Pull request merged**